src/main/java/org/jfrog/hudson/pipeline/common/executors/CollectIssuesExecutor.java

```diff
@@ -56,7 +56,8 @@ private ArtifactoryBuildInfoClientBuilder getBuildInfoClientBuilder(ArtifactoryS
         org.jfrog.hudson.ArtifactoryServer server = Utils.prepareArtifactoryServer(null, pipelineServer);
         CredentialsConfig preferredDeployer = CredentialManager.getPreferredDeployer(new ArtifactoryConfigurator(server), server);
         return server.createArtifactoryClientBuilder(preferredDeployer.provideUsername(build.getParent()),
-                preferredDeployer.providePassword(build.getParent()), server.createProxyConfiguration(Jenkins.getInstance().proxy), new JenkinsBuildInfoLog(listener));
+                preferredDeployer.providePassword(build.getParent()), preferredDeployer.provideAccessToken(build.getParent()),
+                server.createProxyConfiguration(Jenkins.getInstance().proxy), new JenkinsBuildInfoLog(listener));
```

Comment:
The above code is pretty hard to read. I suggested making it a bit less nested or improving the code alignment.